user_docs/en/userGuide.t2t

```diff
@@ -1553,7 +1553,7 @@ Many Windows and controls show a small message (or tooltip) when you move the mo
 ==== Report notifications ====[ObjectPresentationReportNotifications]
 This checkbox, when checked, tells NVDA to report help balloons and toast notifications as they appear.
 - Help Balloons are like tooltips, but are usually larger in size, and are associated with system events such as a network cable being unplugged, or perhaps to alert you about Windows security issues.
-- Toast notifications have been introduced in Windows 10 and appear in the notification center in the system tray, informing about several events (i.e. if an update has been downloaded, a new e-mail arived in your inbox, etc.).
+- Toast notifications have been introduced in Windows 10 and appear in the notification centre in the system tray, informing about several events (i.e. if an update has been downloaded, a new e-mail arrived in your inbox, etc.).

```

Comment:
Microsoft Style Guide indicates the term should be "notification area" (rather than notification centre or other temrs).

Reply:
Link to Style Guide section on notification area: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/notification-area